### DISCUSSION TITLE
Organization not shownig on profile

### DISCUSSION THREAD

**Discussion by MrH4CK3474** [community/community#102668] [Codespaces]
### Select Topic Area

Question

### Body

I have issue with my organization as it was not showing on my profile.

**Comment by ghost** [comment]
Hello. You can follow these steps to troubleshoot the issue:

- Check Membership: Ensure that you are a member of the organization in question. You must be a member of an organization to have it displayed on your profile.

- Privacy Settings: Verify the organization's privacy settings. If the organization has set its visibility to "Private," it will not appear on your profile. Contact the organization's administrators to check and adjust the settings if needed.

- Refresh Profile: Sometimes, it may take a little time for changes to your organization memberships to reflect on your profile. Try refreshing your profile page or logging out and logging back in.

- GitHub Pages: If you are using GitHub Pages to display your profile, ensure that your GitHub Pages repository includes the organization's name in the repository name (e.g., "username.github.io" for your personal profile, or "username.github.io/organization-name" for the organization).

GitHub Support: If the issue persists after checking the above points, consider reaching out to GitHub Support for further assistance. They can help troubleshoot any technical issues related to your profile )))

**Comment by iDevPrit** [comment]
Just go to your organization and visit people section under your organization and make visibility public done now organization are visible on your account.
